websockets/1.0/modules/ROOT/pages/websockets-connector-examples.adoc

```diff
@@ -27,45 +22,44 @@ This app generaates continuous random stock quotes like the following:
 }
 ----
 
-In this example, this is called by the `Quote` type.
+* The `Quote` type identifies the previous mentioned stock quote sample.
+* The Mule app generates approximately 50 quotes with random stock prices that uses at least 5 different tickers, such as CRM, MELI, GOOG, NFLX and APPL.
+* The app also exposes the TLS secure `+wss://localhost:8080/feed+` endpoint that accepts only one connection.
+* The first client must connect to the endpoint successfully.
+* Subsequent clients need to receive a WSS message with the text `"Sorry, spot taken"` after that, the connection is dropped.
```

Comment:
I don't understand this. Why is the connection dropped? Should the other clients not be connected?